### ISSUE TITLE
.EXE for Windows

### ISSUE THREAD

**Issue by JamesTheAwesomeDude** [JamesTheAwesomeDude/mentalDarkChess#10]
there needs to be a canned workflow option for people who aren't on Unix systems or otherwise uber-familiar with the Python runtime

**Comment by JamesTheAwesomeDude** [comment]
changes to README in d5e7c3b564045e7f6de0123eb930fd54e0761423 are close enough for now lol

I'm still open to adding further instructions, just closing as low priority until after #11 is history

**Comment by JamesTheAwesomeDude** [comment]
given the fact that pyinstaller can provide a "merely" 11MiB compressed release, I think we can do way better

at the very least, once any USABLE p2p comms are ready [#4], we must create a tag and then a release with executables for people to use

**Comment by JamesTheAwesomeDude** [comment]
it seems that Proton-GE works great for building executables

![(darkchess_venv) C:\\users\\steamuser\\My Documents\\DarkChess](https://cdn.discordapp.com/attachments/668603669901606922/942282976279023616/unknown.png)

**Comment by JamesTheAwesomeDude** [comment]
> it seems that Proton-GE works great for building executables

ahaha, nope:

![DarkChess-Windows.W1064.exe - Entry Point Not Found: The procedure entry point __wine_dbg_get_channel_flags could not be located in the dynamic link library G:\TEMP\_MEI249402\ucrtbase.dll.](https://media.discordapp.net/attachments/640793365612593152/942299244725956638/Error.PNG?width=400&height=179)

**Comment by JamesTheAwesomeDude** [comment]
Fixed in 9d71795b8b08b2a857fbb03480a9c7675772266c
